itests/integration_test_resource_manager.py

```diff
@@ -29,7 +29,7 @@ def setUpClass(self):
 
             if yarn_endpoint_uri.hostname and yarn_endpoint_uri.port:
                 self.configured = True
-                self.resourceManager = ResourceManager(yarn_endpoint_uri.hostname, yarn_endpoint_uri.port)
+                self.resourceManager = ResourceManager(yarn_endpoint_uri.hostname + ":" + str(yarn_endpoint_uri.port))
```

Comment:
Could you please refactor this to be resource_manager

Reply:
Addressed. Let me know what u think.